package.json

```diff
@@ -17,16 +20,25 @@
     "mongoose": "^6.2.1",
     "morgan": "^1.10.0",
     "nodemon": "^2.0.15",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
Here we have a problem. I checked with the correct command and  only installs this ```{
  "name": "test",
  "private": true,
  "version": "0.0.0",
  "scripts": {
    "dev": "vite",
    "build": "vite build",
    "preview": "vite preview"
  },
  "dependencies": {
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
  },
  "devDependencies": {
    "@vitejs/plugin-react": "^1.0.7",
    "vite": "^2.8.0"
  }
}``` I don't understand why do we have react router and vite as dependencies.